packages/build/webpack/webpack.prod.config.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
   module: {
```

Comment:
How do the changes in this PR impact compilation times of prod builds?

Connect is built after every commit to the teleport repo so I'm mostly worried about that right now + time spent in CI on building the Web UI and Connect.

Reply:
Why does this matter though? I'm not aware of any part of the dev workflow that depends on the results of a build after something is committed to main. What's the difference to you if it takes 1 minute or 10?

Reply:
I remember [Trent pointing out that the macOS builders take a lot of time](https://gravitational.slack.com/archives/C0DF0TPMY/p1655950985331039?thread_ts=1655940010.768399&cid=C0DF0TPMY) so I didn't want to make this even worse if we don't have to.

And when it comes to CI it's best to just keep it as short as possible, I think.

Reply:
I know this is based on just three runs but it looks like the CI times are not affected overall. Linting step takes one minute less but both build steps take two minutes more. When we run all three in parallel this results in no difference in time.

* Current master CI https://console.cloud.google.com/cloud-build/builds/d67c329e-1d5f-41a2-866a-06228714acfc?project=ci-account
* Current master CI https://console.cloud.google.com/cloud-build/builds/174cbd8b-9028-4891-bd1a-c27989149622?project=ci-account
* This PR but all three steps in parallel https://console.cloud.google.com/cloud-build/builds;region=global/3911680d-bd1a-46c9-85d0-280faf69e9ef;step=1;tab=detail?project=ci-account